guides/v2.1/ext-best-practices/tutorials/copy-fieldsets.md

```diff
@@ -53,46 +53,65 @@ The code snippet in the next step uses the name of the fieldset and aspect to sp
 ## Step 3: Copy the fieldset
 {:#step-3}
 
+For copying the fieldset, we'll observe the `sales_model_service_quote_submit_before` event by using the following code in our `etc/events.xml`:
+
+{% highlight xml %}
```

Comment:
Can you change it by ``` xml

Reply:
@osrecio thank you for review. I've adjusted the xml snippets, but if I'm not mistaken, there shouldn't be that space, correct me if I'm wrong.

Reply:
You're right, without spaces, I edited my comment before but you just write your comment hehe. 

Thanks!